src/mod_auth_cas.c

```diff
@@ -2401,32 +2402,35 @@ int cas_match_attribute(const char *const attr_spec, const cas_saml_attr *const
 		 * name and the attr_spec is a tilde (denotes a PCRE match). */
 		else if (!(*attr_c) && (*spec_c) == '~') {
 			const cas_saml_attr_val *val;
-			const char *errorptr;
-			int erroffset;
-			pcre *preg;
+			int errorptr;
+			PCRE2_SIZE erroffset;
+			pcre2_code *preg;
+			uint32_t options = PCRE2_UTF;
```

Comment:
I could be wrong, but it looks like UTF support is compiled by default, but this has to be set to actually use it.  Is there any reason why we wouldn't want it to use Unicode and stick to the first 256 chars of the ASCII table?  This seems like a benefit to non-English users.

Ref. https://www.pcre.org/current/doc/html/pcre2api.html

Reply:
@dhawes I don't want to hold this up.  If the default to UTF for this might cause a delay, I can push an update to remove it and patch it on our end, though I suspect other distros will also want to support UTF here.

Reply:
I tend to agree that supporting UTF is generally a good thing. I can't remember my rationale from last year, but I'd bet it was to just be conservative. I don't have an issue with this. 

Here's the document from the link above for reference:

```
PCRE2_UTF

This option causes PCRE2 to regard both the pattern and the subject strings that are subsequently processed as strings of UTF characters instead of single-code-unit strings. It is available when PCRE2 is built to include Unicode support (which is the default). If Unicode support is not available, the use of this option provokes an error. Details of how PCRE2_UTF changes the behaviour of PCRE2 are given in the [pcre2unicode](https://www.pcre.org/current/doc/html/pcre2unicode.html) page. In particular, note that it changes the way PCRE2_CASELESS handles characters with code points greater than 127.
```

Reply:
Also, apologies to not getting this sooner. We're all merged and good now!